docs/csiaddonsnode.md

```diff
@@ -0,0 +1,17 @@
+# CSIAddonsNode
+
+CSIAddonsNode is a custom resource designed to allow the CSI-Addons Controller(s) to discover CSI-Addons side-cars that are running alongside of CSI-driver components. The information provided in a CSIAddonsNode CR contain details on how to connect to the CSI-Addons side-car.
+
+```yaml
+apiVersion: csiaddons.openshift.io/v1beta1
+kind: CSIAddonsNode
+metadata:
+  name: csiaddonsnode-sample
+spec:
+    driver:
+        name: rbd/cephfs.csi.ceph.com
```

Comment:
```suggestion
  name: csiaddonsnode-sample
spec:
    driver:
        name: rbd/cephfs.csi.ceph.com
        nodeID: node-1
```
Please add nodeID(optional) to identify on which node the side-car is running on.

Reply:
Indeed, we still need to know the `nodeID` so that we can identify which node has a volume attached. The endpoint is not sufficient for that.

Because this is not a driver-list, but a single driver, the `nodeID` probably fits in the `CSIAddonsNodeSpec` directly.

Reply:
@nixpanic  IMO, nodeID should be the part of `CSIAddonsNodeDriver` ad the description of it says that it contains all the information of the CSI Driver existing on the node, which will include nodeID too. Let me know if you still want that to be added in CSIAddonsNodeSpec directly.